src/test/bin/quic_gtest.cpp

```diff
@@ -1967,6 +1967,18 @@ TEST_P(WithAbortiveArgs, AbortiveShutdown) {
     }
 }
 
+TEST_P(WithCancelOnLossArgs, CancelOnLossSend) {
+    TestLoggerT<ParamType> Logger("QuicCancelOnLossSend", GetParam());
+    if (TestingKernelMode) {
+        QUIC_RUN_CANCEL_ON_LOSS_PARAMS Params = {
```

Comment:
I was unable to test this code path. Aligned it with the other IOCTL calls.

Reply:
The automation will take care of it. Thanks!